api/0.1.txt

```diff
@@ -343,6 +343,14 @@ package androidx.os {
 
 package androidx.text {
 
+  public final class SpannableStringKt {
```

Comment:
Don't modify this file

Reply:
I just reversed it. Are we supposed to change only the current.txt?